learning_service/src/test/test_text_preprocessing.py

```diff
@@ -4,8 +4,8 @@
 import numpy as np
 import pandas as pd
 import unittest
-from src.read_data import read_data_from_file
-from src.text_preprocessing import text_process, preprocess_bag_of_words, \
+from learning_service.src.read_data import read_data_from_file
+from learning_service.src.text_preprocessing import text_process, preprocess_bag_of_words, \
```

Comment:
Not sure but importing is a mess in python and not sure if this will work.

Reply:
My IDE was going crazy without the the name of the root module, adding it fixed the problem. I've tested it by running tests in `test_text_preprocessing.py` and everything passed.

![Screenshot from 2022-06-06 22-19-02](https://user-images.githubusercontent.com/4958634/172242026-81ea516c-ce8b-4ac0-b2f3-ed1e2200fe58.png)